app/views/chargeback/_cb_rate_show.html.haml

```diff
@@ -33,7 +33,7 @@
           %td
             = r[:description]
           %td{:align => "right"}
-            = r[:rate]
+            = r[:rate] + " " + r.detail_currency.symbol + " [" + r.detail_currency.code + "]"
```

Comment:
Same here, try to use string interpolation over String#+ for simplicity and less created strings.